utils/responseUtil.ts

```diff
@@ -0,0 +1,56 @@
+import { NextApiResponse } from "next";
+
+/**
+ * @class ResponseUtil: Class for handling responses
+ */
+export class ResponseUtil {
+    /**
+     * Method for returning a status 200 success response
+     * @param res NextApiResponse
+     * @param message Optional message
+     * @param data Optional data
+     */
+    returnSuccess(
+        res: NextApiResponse,
+        message?: string,
+        data?: unknown,
+    ): void {
+        res.status(200).json({ message, data });
+    }
+
+    /**
+     * Method for returning a status 400 bad request response
+     * @param res NextApiResponse
+     * @param message Optional message
+     */
+    returnBadRequest(res: NextApiResponse, message?: string): void {
```

Comment:
`errorMessage` would be a better name

Reply:
also - I think we can make that parameter required and the caller of this function should always pass in an error message

Reply:
this function can be static as it doesn't use any member variables

Reply:
Sounds good to me!

Reply:
done